.github/workflows/test.yml

```diff
@@ -12,7 +12,37 @@ on:
       - main
 
 jobs:
-  run-tests:
+  test:
+    name: Run tests job
+
+    strategy:
+        matrix:
+          store: [memory, badger]
+          namespace: [namespace, none]
+
+    runs-on: ubuntu-latest
+
+    env:
+      CORE_KV_MULTIPLIERS: ${{ matrix.store }},${{ matrix.namespace }}
```

Comment:
question: Why not 2 separate env variables?

Reply:
One is a lot simpler.  Fewer to track/find when reading the code.  A new environment variable per multiplier scales pretty badly IMO.